MapboxVision/src/main/java/com/mapbox/vision/utils/UuidHolder.kt

```diff
@@ -22,18 +21,15 @@ internal interface UuidHolder {
             private set
 
         override fun start() {
-            val backgroundTimestamp = prefs.getLong(BACKGROUND_TIMESTAMP_KEY, 0L)
-            val currentTimestamp = System.currentTimeMillis()
-            if (currentTimestamp - backgroundTimestamp > MAX_BACKGROUND_TO_REFRESH_UUID_MILLIS) {
-                uniqueId = UUID.randomUUID().toString()
-                prefs.edit().putString(UUID_KEY, uniqueId).apply()
-            } else {
-                uniqueId = prefs.getString(UUID_KEY, "")!!
+            if (uniqueId.isBlank()) {
```

Comment:
Do you think we could get rid of start-stop functions and just have single property that inits UUID and stores it to prefs?

Reply:
looks like it's architecture approach. Components are started all all together and stopped as well. No?